plugin/src/main/java/org/gradle/testretry/internal/config/TestTaskConfigurer.java

```diff
@@ -46,9 +48,34 @@ public static void configureTestTask(Test test, ObjectFactory objectFactory, Pro
 
         test.getInputs().property("retry.failOnPassedAfterRetry", adapter.getFailOnPassedAfterRetryInput());
 
+        Provider<Boolean> isDeactivatedByTestDistributionPlugin = shouldTestRetryPluginBeDeactivated(test, objectFactory, providerFactory);
+        test.getInputs().property("isDeactivatedByTestDistributionPlugin", isDeactivatedByTestDistributionPlugin);
```

Comment:
🦺  Perhaps this should have a corresponding test proving task results are not reused when running with and without test distribution (without changes to the classpath).

Reply:
Good point, I'll do that once #103 is merged

Reply:
Added